dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/model/FromInstruction.java

```diff
@@ -14,6 +14,7 @@
 
     private static final String NAME = "FROM";
     private static final String INVALID_INSTRUCTION_LINE = "You have not provided a valid FROM instruction line.";
+    private static final String NO_DFIU = "no-dfiu";
```

Comment:
Let’s bring this in as a command line param w/ the this you have as default if it is not passed

Reply:
Added support for new command line parameter
parser.addArgument("-x")
                .help("comment to process after FROM instruction for ignoring PR creation, if empty, 'no-dfiu' will be used");

Reply:
Thanks!